src/AutoConstructor.Generator/CodeGenerator.cs

```diff
@@ -1,16 +1,22 @@
+using AutoConstructor.Generator.Core;
+using AutoConstructor.Generator.Models;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
 
 namespace AutoConstructor.Generator;
 
-internal class CodeGenerator
+internal sealed class CodeGenerator
 {
     private MemberDeclarationSyntax? _current;
     private bool _addNullableAnnotation;
     private string? _constructorDocumentationComment;
 
+    internal static readonly string GeneratorVersion = (System.Reflection.Assembly.GetExecutingAssembly()
+        .GetCustomAttributes(typeof(System.Reflection.AssemblyFileVersionAttribute), false)
+        .Single() as System.Reflection.AssemblyFileVersionAttribute)!.Version;
```

Comment:
Can this not be simplified?
```suggestion
    internal static readonly string GeneratorVersion = typeof(CodeGenerator).Assembly.GetName().Version!.ToString();
```